templates/tidy_cron.epp

```diff
@@ -12,7 +12,7 @@ find "$DIR" -type f -ctime +$RETENTION_DAYS -delete
 
 #compress files
 cd "$DIR"
-find . -type f ! -name "*.bz2" > "$DIR.tmp";
-xargs -a "$DIR.tmp" tar -jcf "$DIR/$METRICS_TYPE-$(date +%Y.%m.%d.%H.%M.%S).tar.bz2" 2>/dev/null;
-xargs -a "$DIR.tmp" rm ;
-rm "$DIR.tmp";
+find . -type f -not -name "*.gz" > "$DIR.tmp"
```

Comment:
I haven't tested this, but I suspect this line will need something along the lines of `-o -name "*.bz2"`.  Otherwise customers that have this already in place with bzip2 will end up compressing their existing bz2 files into their new gzip files.

Reply:
Good point! I'm on it.

Reply:
Fixed. Turns out it has to be `-a -not`